chat_agent_commander:
  ros__parameters:
    speech_to_text:
      host: "http://10.0.0.131:8082"
      api_key: ''
      name: ''

    language_model:
      host: "http://10.0.0.131:8083"
      api_key: ''
      name: ''
      # This is only required if requesting from servers, which are not already initialized by this custom prompt.
      initial_prompt_file: /home/user/Work/ROS/github/local/robot-commander/prompts/robot-chat.txt
      initial_prompt_context: "You are Loki, the god of mischief and you occasionally try to deceive the human. You are also a robot being controlled using ROS, and your witty remarks remain mostly within this context."

    text_to_speech:
      host: "http://10.0.0.131:8084"
      api_key: ''
      name: ''
      voice_type: 'en_speaker_1' # this will depend on the used host, make sure to use a valid type that is supported by the server
      # TODO(tts-server): local tts server is only experimental for now, use the pytorch models for best results
      pytorch_model_path: /media/user/data/models/bark/hf/bark/
      use_pytorch: True

ros_agent_commander:
  ros__parameters:
    speech_to_text:
      host: "http://10.0.0.131:8080"
      api_key: ''
      name: ''

    language_model:
      host: "http://10.0.0.131:8081"
      api_key: ''
      name: ''
      grammar_file: /home/user/Work/ROS/github/local/robot-commander/grammars/posestamped.json
      # This is only required if requesting from servers, which are not already initialized by this custom prompt.
      initial_prompt_file: /home/user/Work/ROS/github/local/robot-commander/prompts/ros-publisher.txt
      ros_messages_path: /home/user/Work/ROS/github/local/robot-commander/messages/
      initial_prompt_context: "You are located inside a university campus building, which contains several laboratories that you can explore."
